mobile-first.html

```diff
@@ -0,0 +1,223 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <link rel="stylesheet" href="style.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@100;300&display=swap" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&display=swap" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@100&display=swap" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Inter&display=swap" rel="stylesheet">
+
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Bezalell Portfolio</title>
+  </head>
+  <body>
+    <!-- navbar -->
+    <header>
+      <nav class="navbar">
+        <div class="name-container">
+          <a href="index.html">
+            <h2>Bezalell</h2>
+          </a>
+        </div>
+        <div>
+          <div class="bar1"></div>
+          <div class="bar2"></div>
+          <div class="bar3"></div>
+        </div>
```

Comment:
You header design is good. But I have noticed that the size of the hamburger icon in your design is `width:35` and `height:39` while in Figma design the dimensions are `18x12`.
| Your Design   | FIGMA Design  |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/61361037/220964818-5f7e4d4f-acef-46ab-9186-cc9aad2b12ce.png)| ![image](https://user-images.githubusercontent.com/61361037/220964730-835ed219-1a5f-4fe0-9328-6ba879b48344.png)|

Kindly update the hamburger dimensions.